bedrock/settings/base.py

```diff
@@ -1337,6 +1343,11 @@ def JINJA_CONFIG():
     'firefox_os_mwc_2015_preview': {
         'source_filenames': (
             'js/base/mozilla-modal.js',
+            'js/libs/jquery.waypoints.min.js',
+            'js/libs/jquery.waypoints-sticky.min.js',
+            'js/libs/matchMedia.js',
```

Comment:
`'js/libs/matchMedia.js' is only needed for IE9 and is already included via a conditional comment in the [base template](https://github.com/mozilla/bedrock/blob/master/bedrock/base/templates/base-resp.html#L147) for all pages

I think `'js/libs/matchMedia.addListener.js' also needs to be in a conditional comment, to avoid errors in IE8 and below.